### ISSUE TITLE
ADCP plots DIAGRAM_*_vs_DEPTH colormap variable query

### ISSUE THREAD

**Issue by sspagnol** [aodn/imos-toolbox#537]
Just want to check if DIAGRAM_*_vs_DEPTH plots colourmap is  depth. 

And wonder if say 
- DIAGRAM_CDIR_vs_DEPTH coloured by CSPD
- DIAGRAM_CSPD_vs_DEPTH coloured by CDIR
- DIAGRAM_CDIR_vs_DEPTH coloured by qcflag, possibly to highlight if have some mooring interference.

would be useful?

**Comment by ggalibert** [comment]
> Just want to check if DIAGRAM_*_vs_DEPTH plots colourmap is depth.

No, the colourmap is just defined so that each distinct instrument (or bin for ADCPs) gets a different colour. I thought it'd be more useful than DEPTH since DEPTH is already an axis and also because it enables you to see what is the depth range of data collected per instrument.

> And wonder if say 
> - DIAGRAM_CDIR_vs_DEPTH coloured by CSPD
> - DIAGRAM_CSPD_vs_DEPTH coloured by CDIR
> - DIAGRAM_CDIR_vs_DEPTH coloured by qcflag, possibly to highlight if have some mooring interference.
>
> would be useful?

You tell me! :-) maybe worth exposing to others and see what they think?

**Comment by sspagnol** [comment]
Thanks. Should have looked at the code to confirm :)

Will think about the other plots.
